### ISSUE TITLE
remapping K to i<CR><Esc> causes K to move cursor left 1

### ISSUE THREAD

**Issue by robertkrossa** [VSCodeVim/Vim#923]
I have the following code in my workspace settings. I
```
  "vim.otherModesKeyBindings": [
          {
              "before":["K"],
              "after":["i","<CR>","<Esc>"]
          }
      ],
```
Instead of K inserting a new carriage return like its supposed to, it just moves the cursor left 1.
- VSCode Version: 1.6.1
- VsCodeVim Version: 0.3.7
- OS: osx 10.11.6

**Comment by rltbennett** [comment]
Was "after" supposed to be ["i", "<CR>", "<Esc>"]?

**Comment by johnfn** [comment]
This works now and will be released in 0.4.7.
